src/clarity-angular/data/datagrid/datagrid-column-toggle-title.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+
+@Component({
+    selector: "clr-dg-column-toggle-title",
+    template: `<ng-content></ng-content>`,
```

Comment:
The main value would be to verify that we don't bork this template, otherwise its simply testing content projection. Added a test to both PRs for this.